src/worldstate/include/routines/State.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Created By:  Joel Ahn
+ * Created On:  May 6th, 2018
+ * Description: Superclass detailing the individual nodes
+ *              of the world state finite state machine
+ */
+
+#ifndef PROJECT_ROUTINE_H
+#define PROJECT_ROUTINE_H
+
+#include <ros/ros.h>
+#include <worldstate/StateMsg.h>
+/*
+ * Documentation to Provide
```

Comment:
We should provide this :stuck_out_tongue_winking_eye:

Reply:
Fixed!